interfaces/builtin/shared_memory.go

```diff
@@ -55,11 +58,18 @@ const sharedMemoryBaseDeclarationPlugs = `
 // application or gadget snaps to use the slot much like the content interface.
```

Comment:
could you update this comment to reflect that we also now have an implicit system slot for shared-memory as well?